### DISCUSSION TITLE
A way to pop the video out

### DISCUSSION THREAD

**Discussion by agnanas** [biigle/core#970] [Ideas]
The video processing page of biigle at times feels a bit cramped with the timeline at the bottom and the sidebar to the right. In fairness both of them can be adjusted/collapsed, however I often need to use all three section actively. Would be great if we have a way to pop the video or the timeline and label sections out. That way I can use one screen to maximise my video playback and the other screens to track the timeline and label sections.

**Comment by mzur** [comment]
Thanks for the idea! This is something we actually thought of before: https://github.com/biigle/core/issues/674 Since this is now an official user request, I'll try to have someone look at it to see if it's possible.
